.deploy/dev-fss-teamforeldrepenger.json

```diff
@@ -57,6 +56,14 @@
       "roles": [
         "ekstern"
       ]
+    },
+    {
+      "app": "azure-token-generator",
```

Comment:
Hva slags applikasjon er dette?

Reply:
Brukes for å kunne innhente tokenet fra azure for å kunne teste direkte mot dev. Jeg fjerner den siden AAP ikke har nytte for dette lenger.